cmd/sonobuoy/app/run.go

```diff
@@ -82,11 +83,12 @@ func submitSonobuoyRun(f *genFlags) func(cmd *cobra.Command, args []string) {
 			os.Exit(1)
 		}
 
-		if !f.skipPreflight {
+		if !contains(f.skipPreflight, "true") && !contains(f.skipPreflight, "*") {
```

Comment:
I'm fine leaving `*` as a special case just FYI. In my recommended docs I didnt list it but if you think it reads ok to add another special case, feel free. `...true or * will skip all...`. In typical shells you have to escape the `*` so it can be a bit less intuitive.